cpp/benchmarks/join/join_benchmark.cu

```diff
@@ -68,9 +72,12 @@ static void join_benchmark(benchmark::State& state)
 
     int columns_to_join[] = {0};
 
-    gdf_column* result_table[3];
-    for (int icol = 0; icol < 3; icol++) {
-        result_table[icol] = new gdf_column;
+    const int nresult_cols = build_table.size() + probe_table.size() - 1;
+
+    std::vector<gdf_column*> result_table(nresult_cols);
+
+    for (auto & col_ptr : result_table) {
+        col_ptr = new gdf_column;
```

Comment:
Using a `std::vector<gdf_column>` would eliminate the need to heap allocate the `gdf_column` objects with `new`.

Reply:
Resolved in https://github.com/rapidsai/cudf/pull/2105/commits/8c7fda26090cb7ea0bd7e2b6570aef9f2db8e164.